test/elasticsearch.yml

```diff
@@ -1,68 +0,0 @@
-apiVersion: apps/v1
```

Comment:
Please don't remove this file. It is needed for the `es` Makefile target. This target is still needed to run some of the `examples`.